### DISCUSSION TITLE
A Question Related to Margins

### DISCUSSION THREAD

**Discussion by kmarler18** [tailwindlabs/tailwindcss#10062] [Help] [answered]
Hey gang,

Here's what my page looks like ...

![Screen Shot 2022-12-12 at 3 58 28 PM](https://user-images.githubusercontent.com/96091818/207153280-df59f2eb-03e1-4fe5-b8cb-d7b6b606f8f0.png)

This is what I want it to look like ...

![Screen Shot 2022-12-12 at 4 12 03 PM](https://user-images.githubusercontent.com/96091818/207155754-473b8ebc-a540-41e2-a203-23f83125d54e.png)

Here's my code ...

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <script src="https://cdn.tailwindcss.com"></script>
</head>
<body>
    <header class="h-[42rem]">
        <div class="bg-gray-300 h-full bg-center bg-no-repeat bg-cover">
            <nav class="bg-white mt-4 mx-4">
                Test Content
            </nav>
        </div>
    </header>
</body>
</html>
```

When applying mt-4 to the nav element I expected the page to look like my 2nd screenshot, but instead, it made it look as though I have a top margin applied to the body element. I've included a screenshot to the Chrome Developer Tools to hopefully add more clarity. How can I make it so that the top margin of the nav element is applied from its parent div element and not from the html element? 

![Screen Shot 2022-12-12 at 4 20 49 PM](https://user-images.githubusercontent.com/96091818/207157162-6407113a-274e-4981-8b31-cbe03f2b8d3f.png)

**Comment by mohamedpppppppppp** [comment]
fdgdfgd

**Comment by wongjn** [comment, marked as answer]
This is happening due to [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing). To work around this, you could try applying `padding-top` via `pt-4` to the `<div>` element instead of `mt-4` on the `<nav>` element.

**Reply by kmarler18** [reply]
Hey Justin. Thank you so much. You're 2 for 2 on my questions!

Just out of curiosity, are you a web developer by profession?
